spacy_llm/tasks/ner.py

```diff
@@ -57,48 +97,35 @@ def _unique(items: Iterable[str]) -> Iterable[str]:
 
 
 @registry.llm_tasks("spacy.NER.v1")
-class NERTask:
-    _TEMPLATE_STR = """
-From the text below, extract the following entities in the following format:
-{# whitespace #}
-{%- for label in labels -%}
-{{ label }}: <comma delimited list of strings>
-{# whitespace #}
-{%- endfor -%}
-{# whitespace #}
-{%- if examples -%}
-{# whitespace #}
-Below are some examples (only use these as a guide):
-{# whitespace #}
-{# whitespace #}
-{%- for example in examples -%}
-{# whitespace #}
-Text:
-'''
-{{ example.text }}
-'''
-{# whitespace #}
-{%- for label, substrings in example.entities.items() -%}
-{{ label }}: {{ ', '.join(substrings) }}
-{# whitespace #}
-{%- endfor -%}
-{# whitespace #}
-{# whitespace #}
-{%- endfor -%}
-{%- endif -%}
-{# whitespace #}
-Here is the text that needs labeling:
-{# whitespace #}
-Text:
-'''
-{{ text }}
-'''
-    """
+def make_ner_task(
+    labels: Union[str, Iterable[str]],
+    template: str = _DEFAULT_NER_TEMPLATE,
```

Comment:
Might be out of scope for this PR, I wonder about the use-case where (i) I want to write my own prompt template and (ii) I want to do it in the config file. It might be a bit unruly if it's just a single line in the config. I think this is where something like a `spacy.JinjaReader.v1`  comes to play? (cc: @rmitsch ) 

Something like...

```ini
[components.llm.task]
@llm_tasks = "spacy.NER.v1"
labels = "PER,ORG,LOC"
...

[components.llm.task.template]
@misc = "spacy.JinjaReader.v1"
path = "path/to/my/template.jinja2"
```

Reply:
Agree with you on both :wink: Having the prompt template in the config file will be ugly, but let's do that in a different PR.

Reply:
Continued in https://github.com/explosion/spacy-llm/pull/95.